assets/design.md

```diff
@@ -140,6 +151,31 @@ The `decorator` looks pretty similar to the `configurator` the only difference b
 
 #### Why do we need both Configurators and Decorators?
```

Comment:
Should this reflect the name change?

Reply:
No, this is meant to be part of the remove service catalog commit.